roles/privacyidea/handlers/main.yml

```diff
@@ -1,6 +1,28 @@
 ---
-  - name: restart sshd
-    service: name=sshd state=restarted
+- name: restart nginx
+  service:
+    name: nginx
+    state: restarted
+    enabled: yes
+
+- name: restart uwsgi
+  systemd:
```

Comment:
i do not think so, since we use service: more often then not and systemd is only necessarry in special usecases while service: is more generic, i will switch them to serrvice: